src/SFA.DAS.ApprenticeAccounts.UnitTests/Application/Commands/MyApprenticeship/CreateMyApprenticeshipCommandValidatorTests/CreateMyApprenticeshipCommandValidatorTests.cs

```diff
@@ -0,0 +1,201 @@
+﻿using FluentAssertions;
+using FluentValidation.TestHelper;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApprenticeAccounts.Application.Commands.CreateMyApprenticeCommand;
+using SFA.DAS.ApprenticeAccounts.Data;
+using SFA.DAS.ApprenticeAccounts.Data.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mail;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeAccounts.UnitTests.Application.Commands.MyApprenticeship.CreateMyApprenticeshipCommandValidatorTests;
+
+[TestFixture]
+public class CreateMyApprenticeshipCommandValidatorTests
+{
+    private static readonly Random random = new Random();
+    private Mock<IApprenticeContext> _mockApprenticeContext = new Mock<IApprenticeContext>();
+    private Mock<IMyApprenticeshipContext> _mockMyApprenticeshipContext = new Mock<IMyApprenticeshipContext>();
+
+    [TestCase(0, true)]
+    [TestCase(1, true)]
+    [TestCase(200, true)]
+    [TestCase(201, false)]
+    public async Task EmployerName_Validation(int lengthOfString, bool isValid)
+    {
+        var employerName = RandomString(lengthOfString);
+        var command = new CreateMyApprenticeshipCommand { EmployerName = employerName};
+
+        var sut = GetValidator();
+
+        var result = await sut.TestValidateAsync(command);
+        if (isValid)
+            result.ShouldNotHaveValidationErrorFor(c => c.EmployerName);
+        else
+        {
+            result.ShouldHaveValidationErrorFor(c => c.EmployerName);
+            result.Errors[0].ErrorMessage.Should().Be(CreateMyApprenticeshipCommandValidator.EmployerNameTooLong);
```

Comment:
Can be simplified with result.ShouldHaveValidationErrorFor(c => c.EmployerName).WithMessage(CreateMyApprenticeshipCommandValidator.EmployerNameTooLong)